cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -13,7 +13,7 @@ class HttpCache
   # A map from script name to script level URL pattern.
   CACHED_SCRIPTS_MAP = {
     'starwars' => '/s/starwars/stage/1/puzzle/*',
```

Comment:
Great suggestion.  Made that change, below is the resulting map so you can see the resulting map is unchanged.

irb(main):003:0> pp HttpCache::CACHED_SCRIPTS_MAP
{"hourofcode"=>"/hoc/*",
 "starwars"=>"/s/starwars/stage/1/puzzle/*",
 "starwarsblocks"=>"/s/starwarsblocks/stage/1/puzzle/*",
 "mc"=>"/s/mc/stage/1/puzzle/*",
 "frozen"=>"/s/frozen/stage/1/puzzle/*",
 "gumball"=>"/s/gumball/stage/1/puzzle/*"}